tests/test_api_circuit_breaker.py

```diff
@@ -75,7 +75,7 @@ def test_circuit_breaker_404(breaker_test, disable_redis):
             response = client.get(url="http://localhost/v1/pois/osm:way:7777777?lang=es")
 
         assert breaker_test.current_state == "closed"
-        assert len(rsps.calls) == 4
+        assert len(rsps.calls) == 8
```

Comment:
Why do we need these changes of number of calls, here and in `test_rate_limiter.py` and `test_wiki_ES.py`?

Reply:
Now the API returns two blocks with content with Wikipedia API which doubles the calls to the API. In practice Idunn uses an internal cache to avoid those extra calls, but it is disabled for this tests as the circuit breaker (which is being tested) acts behind this cache.

This is a bit messy and there may be a way to refactor things a little (to rely less on the cache), but I think we could just not bother too much about it and just remove the Wikipedia block as soon as this new block is used by erdapfel in production.